crates/bevy_solari/src/scene/binder.rs

```diff
@@ -190,6 +190,7 @@ pub fn prepare_raytracing_scene_bindings(
             vertex_buffer_offset: vertex_slice.range.start,
             index_buffer_id,
             index_buffer_offset: index_slice.range.start,
+            triangle_count: (index_slice.range.len() / 3) as u32,
```

Comment:
I don't love adding this extra field, it's not longer 128 bits :(

And the triangle count already exists in the light source structure, it's just you need this for ray traces, since your not directly sampling the lights. Hmm. Lets talk about this over discord more.